frontend/lib/src/components/widgets/Checkbox/Checkbox.tsx

```diff
@@ -252,10 +252,10 @@ class Checkbox extends React.PureComponent<Props, State> {
                   // use the long-hand version, which means we can't use the
                   // shorthand names here as if we do we'll end up with warn
                   // logs spamming us every time a checkbox is rendered.
-                  borderLeftWidth: "1px",
-                  borderRightWidth: "1px",
-                  borderTopWidth: "1px",
-                  borderBottomWidth: "1px",
+                  borderLeftWidth: theme.sizes.borderWidth,
+                  borderRightWidth: theme.sizes.borderWidth,
+                  borderTopWidth: theme.sizes.borderWidth,
+                  borderBottomWidth: theme.sizes.borderWidth,
```

Comment:
Might just benefit from destructuring theme to get `sizes` or borderWidth directly?

Reply:
Update 👍